test/profiler/test_profiler.py

```diff
@@ -958,6 +958,8 @@ def check_trace(fname):
 
         if torch.cuda.is_available():
             with TemporaryFileName(mode="w+") as fname:
+                if is_gfx94x_arch:
```

Comment:
Need to call the function using `()` similar to https://github.com/ROCm/pytorch/pull/1364/files#diff-57c0f6a344227861850f9d31387ff82cb089898f7472542060349544dade7875R204